protos/cerbos/cloud/bundle/v2/bundle.proto

```diff
@@ -76,6 +100,12 @@ message WatchBundleRequest {
       (buf.validate.field).timestamp.within = {seconds: 60}
     ];
     string active_bundle_id = 2 [(buf.validate.field).string.len = 16];
+    optional BundleType bundle_type = 3 [(buf.validate.field).enum = {
```

Comment:
Does this validation actually make the field required?

Should we have `defined_only` instead here?